moneymarket_account.rb

```diff
@@ -0,0 +1,54 @@
+module Bank
+
+require 'csv'
+
+  class MoneyMarket < Account
+
+    # @@min_balance = 10000
+
+    attr_accessor :transactions
+
+    def initialize(account_id, balance, datetime_open, owner = nil)
+      super
+      #max of 6 transactions (deposit or withdrawal)
+      #initial balance !< $10000 - will raise ArgumentError
+      @min_balance = 10000
+      @transactions = 0
+      @min_balance_fee = 100
+    end
+
+  # def transactions()
+  #   @transactions += 1
+  # end
+
+    def withdraw(withdraw_amount)
+      if (@balance - withdraw_amount) < @min_balance && @transactions < 6
+        @balance -= (@min_balance_fee)
+        puts "A $100 fee is imposed for going below $10000"
```

Comment:
Since you have variables that store these variables, I would use the variables in this error message b/c if the values changed, then you wouldn't need to change this message